infra/cifuzz/cifuzz.py

```diff
@@ -201,6 +216,46 @@ def run_fuzzers(fuzz_seconds, workspace):
   return True, False
 
 
+def download_latest_corpus(project_name, out_dir, target):
+  """Downloads the newest OSS-Fuzz backup corpus from google cloud.
+
+  Args:
+    project_name: The name of the projects backup to download.
+    out_dir: The location to place the download.
+    fuzz_target: The fuzz_target's corpus to be downloaded.
+
+  Returns:
+    The local path to to corpus or None if download failed.
+  """
+  if not helper.check_project_exists(project_name):
+    logging.error('Project %s is not a valid OSS-Fuzz project.', project_name)
+    return None
+  if not os.path.exists(out_dir):
+    logging.error('Out directory %s does not exist.', out_dir)
+    return None
+  corpus_dir = os.path.join(out_dir, 'corpus', target)
+  os.makedirs(corpus_dir, exist_ok=True)
+
+  http_link = 'https://storage.googleapis.com/{0}-backup.clusterfuzz-external' \
+    '.appspot.com/corpus/libFuzzer/{0}_{1}/{2}.zip'
+  current_date = datetime.datetime.now()
+  for day_diff in range(90, 100):
```

Comment:
sorry if we didn't make it clear, you don't need to guess these days, just look for `public.zip` archive. I've landed this change in https://github.com/google/clusterfuzz/pull/1439 and am deploying it right now

Reply:
and I've just run the cron task, the `public.zip` backups are now populated:

```
$ gsutil ls gs://yara-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/*/public.zipgs://yara-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/yara_dex_fuzzer/public.zip
gs://yara-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/yara_dotnet_fuzzer/public.zip
gs://yara-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/yara_elf_fuzzer/public.zip
gs://yara-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/yara_macho_fuzzer/public.zip
gs://yara-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/yara_pe_fuzzer/public.zip
gs://yara-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/yara_rules_fuzzer/public.zip
```

Reply:
Fixed! Thanks for that!